cr-db/README.md

```diff
@@ -11,6 +11,7 @@ Instructions
 
 1. Fork the repo into a private repo.
 2. Create a virtualenv for this project and install the cr-api and cr-db packages into your environment.
+3. pip install -r additional_requirements.txt
 3. Modify the cr-db package to complete the task, the code is commented with task items.
```

Comment:
I might instead change the setup.py to have a test_requires clause, but this is fine.  The enumeration is an abomination tho 😉 .

Reply:
solved in e7dc6ee84950b57c5d0ee39fce9d599a0694f19a